examples/stm32/f1/stm32-pill/fancyblink/README.md

```diff
@@ -0,0 +1,8 @@
+# README
+
+This is small LED blinking example program using libopencm3.
+
+It's intended for the ST STM32-based
+[Olimex STM32-H103 eval board](http://olimex.com/dev/stm32-h103.html).
```

Comment:
No it's not :)  If you're going to copy and paste like this, you need to take care to fix up things like this afterwords.